modules/video_input/video_input_worker.py

```diff
@@ -2,30 +2,56 @@
 Gets images from the camera.
 """
 
+import os
+import pathlib
 import time
 
 from utilities.workers import queue_proxy_wrapper
 from utilities.workers import worker_controller
 from . import video_input
+from ..common.modules.camera import camera_factory
+from ..common.modules.camera import camera_opencv
+from ..common.modules.camera import camera_picamera2
+from ..common.modules.logger import logger
 
 
 def video_input_worker(
-    camera_name: "int | str",
-    period: float,
-    save_name: str,
+    period: int,
```

Comment:
Move period with the worker parameters. Also should be float.

Reply:
It currently is a worker parameter? I changed it to float

Reply:
I meant immediately above `output_queue` . Then the parameters for the class object and the worker are grouped separately.